### ISSUE TITLE
"Process crashed" popup and log entries

### ISSUE THREAD

**Issue by morsedl** [hluk/CopyQ#1007]
Issue #881 is not resolved on Ubuntu 16.04.04 64 bit. I just upgraded to CopyQ 3.5.0, restarted it and the following error is still occurring in a popup window:

```
! Command "copyq provideSelection"
Error: Process crashed
1. copyq provideSelection

```

**Comment by hluk** [comment]
I this still an issue with new version?

**Comment by morsedl** [comment]
Hi Lukas,

**Synposis / TL;DR**

I'm certain I've not encountered the error on one machine (home) since it was updated to 3.9.2 on September 13th. I'm certain I have encountered at another machine (work) as recently as two days ago, but I only just now updated it from 3.8.0 to 3.9.2. Should I encounter the error again on either machine, I'll let you know.

**Version and Error History on Two Machines**

I use CopyQ primarily on my work and home workstations, and my source for installing and upgrading it is your hluk/copyq PPA for Ubuntu. For stability reasons I only apply updates a few times a year, especially with my workstation at work.

On my home machine, apt upgraded CopyQ from 3.9.0 to 3.9.2 on 2019-09-13. Version 3.9.0 was installed 2019-07-15. This may be the original install date, as I used to use glipper and I seem to recall being slow to switch over to CopyQ at home after having switched to it at work months prior.

Regardless, since 3.9.2 (Sep. 13th), I'm certain I that I have _not_ seen this error message at home. I want to say I _did_ still encounter it with 3.9.0, but I cannot say for certain. This is because I keep both my home and work machines so nearly-identically configured (including visual appearance) that I have no memory of seeing the error message in which I distinguish where I was / which machine I was using.

```bash
morse@home ~$ zcat -f /var/log/dpkg.log.* | grep 'status installed.*copyq' | sort -n
2019-07-15 20:47:36 status installed copyq:amd64 3.9.0~xenial
2019-09-13 20:12:41 status installed copyq:amd64 3.9.2~xenial
```
At work I've been on 3.8.0 since May, and I definitely encountered this error multiple times on Friday (i.e., two days ago). I just updated to 3.9.2 on my work machine, and I will let you know if I encounter the error again (either machine).

```bash
morse@work ~$ zcat -f /var/log/dpkg.log.* | grep 'status installed.*copyq' | sort -n
2018-10-02 14:33:39 status installed copyq:amd64 3.4.0-1
2018-10-02 14:34:34 status installed copyq:amd64 3.4.0-1
2018-10-02 14:34:56 status installed copyq:amd64 3.6.1~xenial
2018-10-02 14:35:20 status installed copyq:amd64 3.6.1~xenial
2018-10-02 14:36:40 status installed copyq:amd64 3.6.1~xenial
2019-05-09 19:09:31 status installed copyq:amd64 3.8.0~xenial
```

I hope this helps, and thanks again for such an incredibly useful and well-implemented utility. Switching to CopyQ has been tranformative for me, making all the other clipboard managers like glibber and ditto I've used over the years pale in comparison. Well done, sir! :-)

Cheers,
Doug
